### DISCUSSION TITLE
Way to make windows not hide behind sketchybar?

### DISCUSSION THREAD

**Discussion by ArchDud3** [FelixKratz/SketchyBar#387] [Q&A]
Hey all, I've recently started using sketchybar, and I've been wondering if there was a way to make the window bar not hide behind sketchybar, but just below it. 
Thanks, 
ArchDud3

**Comment by FelixKratz** [comment]
That is typically the job of a window manager. If you, for example, use yabai you can disable windows from being placed below sketchybar.

I have not found a way to tell the macOS windowserver to stop placing windows below sketchybar and "reserve" the screen area for sketchybar exclusively. This is why the window manager has to do this job instead, as it knows about all windows of all applications and can forbid them from being placed in the screen region where sketchybar resides.

If someone is interested in doing some research on this, I would start by figuring out how the window server reserves the screen space for the regular menu bar, then trying to emulate this in sketchybar. However, I would be surprised if this was actually possible with basic user permissions.

**Reply by Jinx-FX** [reply]
When I maximize a window, the window will always cover the bar or be covered by the window.
So if I want to avoid it, I may use yabai to solve it. Right?🧐
